### PR TITLE
Issue 60 helm charts

### PR DESCRIPTION
**Pull request by eladiw**

### CODE REVIEW

File: sampleapp/charts/sampleapp/values.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
```
Comment (itye-msft) [inline sampleapp/charts/sampleapp/values.yaml]: Please remove unnecessary comments

File: sampleapp/charts/sampleapp/values.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+replicaCount: 1
+
+image:
+  repository: agogosacr.azurecr.io/agogo-sample
+  tag: v1
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
```
Comment (itye-msft) [inline sampleapp/charts/sampleapp/values.yaml]: @eladiw 
Can you add comments what each value stands for, and possible values. Give examples. 
Also please add it to MD file is possible.
Please do so for the rest of the values in this file.
Thanks

File: sampleapp/charts/sampleapp/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for the sample app
+name: sampleapp
```
Comment (itye-msft) [inline sampleapp/charts/sampleapp/Chart.yaml]: @eladiw 
Can you please rename the chart to a more meaningful name other than sampleapp?
The chart name should help to imply what it does.

Comment (itye-msft) [reply sampleapp/charts/sampleapp/Chart.yaml]: Also the description.

File: sampleapp/charts/sampleapp/values.yaml (hunk 1 above)
Comment (eladiw) [reply sampleapp/charts/sampleapp/values.yaml]: 👍

File: sampleapp/charts/sampleapp/Chart.yaml (hunk 3 above)
Comment (eladiw) [reply sampleapp/charts/sampleapp/Chart.yaml]: 👍

File: sampleapp/charts/sampleapp/values.yaml (hunk 2 above)
Comment (eladiw) [reply sampleapp/charts/sampleapp/values.yaml]: This is a regular standard (and autogenerated) values.yaml file.
I dont think its needed.

**Comment by eladiw** [comment]
@itye-msft I have incorporated the requested changes (most of them)

File: sampleapp/charts/sampleapp/values.yaml (hunk 2 above)
Comment (itye-msft) [reply sampleapp/charts/sampleapp/values.yaml]: @eladiw 
What `nameOverride` means?
Can this be documented?

File: sampleapp/charts/sampleapp/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A simple app that receives transforms and sends the data
+name: simpletransform
```
Comment (itye-msft) [inline sampleapp/charts/sampleapp/Chart.yaml]: @eladiw 
Please rename to `sample-datapipeline-app`

**Review by itye-msft — APPROVED** [review]

**Pull request merged**